parts/appendices/B.fodt

```diff
@@ -13551,10 +13668,166 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">The simulator now avoids computing the default number of domains based on a specific target number of cells per domain if the user choses a specific number of NLDD domains (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5579" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5579</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">The simulator now avoids doing unnecessary updates in the numerical aquifer calculations (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5632" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5632</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Previously the THP was set to zero if a well was stopped or set to a zero rate (well or group). The THP will now be calculated provided a VFP table is active (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5634" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5634</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Consistency checks have been added for three-point saturation scaling SCALECRS=YES (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5573" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5573</text:a>,<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5640" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5640</text:a>). These checks collectively guarantee a mobile displacing oil saturation in the two phase gas-oil and oil-water systems. </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Support has been added for Python &gt;= 3.8, whilst support has been maintained for Python 3.6 and 3.7 (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5644" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5644</text:a>). </text:p>
```

Comment:
We might want to skip this. 
Said PR just changes the dependencies for the OPM pypi packages to support also newer versions. Version 3.6-3.12 seem supported